### DISCUSSION TITLE
Recommended workflow for tracking a tuning job on a sklearn model

### DISCUSSION THREAD

**Discussion by krishnashanker-amt** [aws/sagemaker-python-sdk#2453] [Help]
Hello!

I'm tuning a sklearn model (by providing a training script to sagemaker.sklearn.estimator.SKLearn) with sagemaker.tuner.HyperparameterTuner (mostly similar to the logic followed [here](https://github.com/aws/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/scikit_learn_randomforest/Sklearn_on_SageMaker_end2end.ipynb) ) . 

I'm not sure how to track a tuning job setup by this process using Sagemaker Experiments (I know we can track a job in the console). I'm able to link all trials created by this tuning job to an experiment AFTER the tuning job completes using the logic followed [here](https://github.com/aws/amazon-sagemaker-examples/tree/master/sagemaker-experiments/associate-hyper-parameter-tuning-job). The `fit()` of HyperparameterTuner does not support `experiment_config` . I've also tried setting up a tracker in the training script (using `my_tracker = Tracker.load()`) . This method creates a trial for each training job, but `my_tracker.log_metric(metric_name='test_metric', value=0.4452)` does not work (no metric is logged in the trial component from tracker) . 

What am I missing? How can I track a tuning job with custom metrics using sagemaker-python-sdk and Sagemaker Experiments?
